src/component/images.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+class Images extends React.Component {
+   
+
+    render(){
+        return(
+            <React.Fragment>
+                <div className="lesImages">
```

Comment:
@malvena15 What would be the alternative to `node.appendChild` which would be more inline with the React component-based lifecycle?